src/lib/utils/getHighlightFromSiblings.ts

```diff
@@ -1,9 +1,13 @@
 import unescape from './unescape';
 import { HighlightedParts } from '../../types';
 
-const getHighlightFromSiblings = (parts: HighlightedParts[], i: number) => {
+const isAlphanumeric = new RegExp(/\w/i);
```

Comment:
now this should be called `hasAlphanumeric` I think, since it's only checking the presence, not whether it's fully alphanumeric 👍

Reply:
Yep that make sense 🤔